AEPIdentity/Tests/IdentityTests.swift

```diff
@@ -307,6 +307,81 @@ class IdentityTests: XCTestCase {
         XCTAssertTrue(mockNetworkService.connectAsyncCalled) // network request for opt-out hit should have been sent
     }
 
+    /// Tests Identity Extension sends an opt out request to default demdex server when configuration experience.server has empty string value
+    func testAudienceResponseEventOptOutHitSentIsFalse_ExperienceServerURLEmpty() {
+        // setup
+        var props = IdentityProperties()
+        props.ecid = ECID()
+        props.saveToPersistence()
+        identity = Identity(runtime: mockRuntime)
+        identity.onRegistered()
+
+        let testOrgId = "testOrgId"
+        let configData = [IdentityConstants.Configuration.GLOBAL_CONFIG_PRIVACY: PrivacyStatus.optedOut.rawValue, IdentityConstants.Configuration.EXPERIENCE_CLOUD_ORGID: testOrgId, IdentityConstants.Configuration.EXPERIENCE_CLOUD_SERVER: ""] as [String: Any]
+        let audienceResponseEventData = [IdentityConstants.Audience.OPTED_OUT_HIT_SENT: false]
+        let event = Event(name: "Test Audience response", type: EventType.audienceManager, source: EventSource.responseContent, data: audienceResponseEventData)
+        mockRuntime.simulateSharedState(extensionName: IdentityConstants.SharedStateKeys.CONFIGURATION, event: event, data: (configData, .set))
+        let _ = identity.readyForEvent(event)
+
+        // test
+        mockRuntime.simulateComingEvent(event: event)
+
+        // verify network request is sent
+        let mockNetworkService = ServiceProvider.shared.networkService as! MockNetworkServiceOverrider
+        let optOutHitHost = mockNetworkService.connectAsyncCalledWithNetworkRequest?.url.host // network request for opt-out hit should have been sent
+        XCTAssertTrue(optOutHitHost!.contains("dpm.demdex.net"))
```

Comment:
Let's unwrap `optOutHitHost` so this test won't crash in the case where the network call doesn't actually get sent.  Maybe just use a guard and if the host is nil then `XCTFail()` the test?